app/views/templates/applicationTemplate.scala.html

```diff
@@ -28,17 +28,17 @@
 
 
 @createLink(path: String)(implicit req: play.api.mvc.Request[_]) = @{
-    val protocol = if(req.secure) "https" else "http"
-    s"$protocol://${req.host}${routes.Assets.versioned(path)}"
+val protocol = if(req.secure) "https" else "http"
+s"$protocol://${req.host}${routes.Assets.versioned(path)}"
 }
 
 @sectionHeading(headerId: String) = {
-    <h2 id=@headerId>@messages(headerId)</h2>
+<h2 id=@headerId>@messages(headerId)</h2>
 }
 
 <html>
-    <head>
-        <style>
+<head>
+    <style>
                 body {
                     font-size: 21px;
                     font-family: "nta", Arial, sans-serif;
```

Comment:
Is there a reason we have the styling within the view?

Reply:
This is the page used to drive the PDF generation so all of the assets need to be in the source I think